### ISSUE TITLE
Add support for toctree

### ISSUE THREAD

**Issue by Akasurde** [mgedmin/restview#35]
I have sphinx-documentation for project which contains rst file which has links to all other document. This is achieved by "toctree" directive of sphinx. 

If you run restview with such rst file which contains toctree directive, restview fails to render document.
It would be good to add support for toctree directive which will make all other rst document as hyperlinks.

I am using 
Fedora 22
restview 2.4.0 
Python 2.7

I would love to contribute on this feature.

**Comment by Akasurde** [comment]
restview internally uses docutils. [Docutils](http://docutils.sourceforge.net/) doesn't support `toctree`. 

Please search `TOC` this [todo items list](http://docutils.sourceforge.net/docs/dev/todo.html)

Closing issue on restview for now.

**Comment by mgedmin** [comment]
Yeah, it'd be nice if restview had an explicit --sphinx mode, but someone would have so submit a pull request, since I'm not likely to find time/energy to implement it myself.

I myself am using https://pypi.python.org/pypi/livereload to preview Sphinx documents.

**Comment by Akasurde** [comment]
@mgedmin if you can help me in design part for `--sphinx` then I can implement it.

**Comment by mgedmin** [comment]
Sorry, I've no ideas about a possible design for this.

**Comment by gamesbook** [comment]
It would be great if restview supported linking and cross-referencing via Sphinx's `:ref:` and `:doc:`
